diff --git a/node_modules/@types/js-yaml/index.d.ts b/node_modules/@types/js-yaml/index.d.ts
index 8dfd30d..654e1f3 100644
--- a/node_modules/@types/js-yaml/index.d.ts
+++ b/node_modules/@types/js-yaml/index.d.ts
@@ -8,7 +8,7 @@
 
 export as namespace jsyaml;
 
-export function safeLoad(str: string, opts?: LoadOptions): string | object | undefined;
+export function safeLoad(str: string, opts?: LoadOptions): string | { [key: string]: any } | undefined;
 export function load(str: string, opts?: LoadOptions): any;
 
 export class Type {
